.changeset/popular-lies-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/stash": patch
+---
+
+Consolidated how state changes are applied and subscribers notified.
```

Comment:
```suggestion
Consolidated how state changes are applied and subscribers notified. Stash subscribers now receive an ordered list of state updates rather than an object.
```